pointerevents/extension/pointerevent_constructor.html

```diff
@@ -17,36 +17,79 @@
             var target0 = document.getElementById("target0");
             // set values for non-default constructor
             on_event(target0, "pointermove", this.step_func(function(event) {
-                generate_tests(assert_equals, [
-                    ["getCoalescedEvents().length", event.getCoalescedEvents().length, 2],
-                    ["event.target", event.target, target0],
```

Comment:
Right, I got that; just wondering why we have `event.target` tested here and seemingly otherwise have them tested in manual tests.